cypress/integration/calls.ts

```diff
@@ -223,15 +223,10 @@ context('Calls tests', () => {
   it('A user-officer should be able to assign instrument/s to a call', () => {
     cy.login('officer');
 
-    cy.contains('Instruments').click();
-    cy.contains('Create').click();
-    cy.get('#name').type(instrumentAssignedToCall.name);
-    cy.get('#shortCode').type(instrumentAssignedToCall.shortCode);
-    cy.get('#description').type(instrumentAssignedToCall.description);
-    cy.get('[data-cy="submit"]').click();
-
-    cy.notification({ variant: 'success', text: 'successfully' });
+    cy.contains('People').click();
+    cy.addScientistRoleToUser('Carl');
```

Comment:
Very minor, but would be good to refactor 'Carl' into a variable, like you've done in `scheduler.ts`:
```ts
const scientist = 'Carl';
cy.addScientistRoleToUser(scientist);
```